CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Release History
 
+### 1.5.0-beta.1 (Unreleased)
+#### Key Bug Fixes
```

Comment:
Can you also please add changelog for this bug fix - https://github.com/microsoft/kafka-connect-cosmosdb/pull/466

Reply:
added